crates/api_models/src/payouts.rs

```diff
@@ -205,110 +234,14 @@ pub struct Bank {
     /// Bank city
     #[schema(value_type = String, example = "California")]
     pub bank_city: String,
-}
 
-#[cfg(feature = "payouts")]
-impl Default for Bank {
-    fn default() -> Self {
-        Self {
-            bank_account_number: Some("000123456".to_string()),
-            bank_routing_number: Some("110000000".to_string()),
-            iban: None,
-            bic: None,
-            bank_sort_code: None,
-            blz: None,
-            bank_transit_number: None,
-            bank_name: "Deutsche Bank".to_string(),
-            bank_country_code: api_enums::CountryAlpha2::US,
-            bank_city: "California".to_string(),
-        }
-    }
-}
+    /// International Bank Account Number (iban) - used in many countries for identifying a bank along with it's customer.
+    #[schema(value_type = String, example = "DE89370400440532013000")]
+    pub iban: String,
```

Comment:
mask it with secret

Reply:
Resolved in a separate commit - https://github.com/juspay/hyperswitch/pull/1384/commits/ac41daafceda596d06c428eb4b15eb15fadd337b